schema.graphql

```diff
@@ -67,3 +48,35 @@ type UniquesTransfer @entity {
   timestamp: Float @index
   success: Boolean
 }
+
+type Pot @entity {
+  id: ID!
+  potId: Float! @index 
+  owner: String! @index
+  feeQuota: BigInt!
+  reserveQuota: BigInt!
```

Comment:
`Quota` should be able to show two fields: `limit` and `balance`.

Reply:
I imagine we'd want to know both the used quota and the available quota about the users. So perhaps it's worth adding both.

Reply:
Update to registry Limit and Balance for feeQuota and reserveQuota